src/components/FeedbackPage.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import Button from '@material-ui/core/Button';
+
+const choices = [
+  {
+    value: 'complaint',
+    label: 'Complaint',
+  },
+  {
+    value: 'suggestion',
+    label: 'Suggestion',
+  },
+  {
+    value: 'request',
+    label: 'Request',
+  },
+];
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    '& .MuiTextField-root': {
+      margin: theme.spacing(1),
+      marginRight: theme.spacing(1),
+      width: '25ch',
+    },
+  },
+}));
```

Comment:
```suggestion
      width: '30ch',
      marginLeft: 20
    },
  },
    input: {
    display: 'flex',
    flexDirection: 'row'
  }
}));
```

Just a small styling adjustment